charts/py-app/templates/static-service.yaml

```
@@ -0,0 +1,18 @@
+{{- if .Values.service.enabled -}}
```

I guess it should be:
```suggestion
{{- if and .Values.service.enabled .Values.service.staticServiceName -}}
```

---

yeah, I thought about that and came up with this :point_down: 
https://github.com/Intreecom/charts/pull/10/files#diff-bbeba371c8952b1e5ca054476d487c190485e8cb72d9d02124f5d56ba719117b

should we have the condition with `and` in manifest file or `default` in helper file ?

---

Oh, I see. So it actually create a name implicitly and also it means that static server will be generated for any project. Sounds fine, but could be optional, imo.

---

yes, that way it's simpler.
added the condition to create static-service